tests/Licenses/LicensesClientTest.php

```diff
@@ -838,4 +839,214 @@ public function testFindWithNullCurrency(): void
+            ->method('request')
+            ->with(
+                'get',
+                'https://www.test.com/licenses/XSP1234/configs',
+                [
+                    'headers' => [
+                        'apiKey' => '123456',
+                    ],
+                ]
+            )
+            ->willReturn(new Response(200, [], 'OK USA'));
+
+        $this->client->getConfigsRaw('XSP1234');
+    }
+
+    /**
+     * @throws EntityValidationException
+     * @throws GuzzleException
+     * @throws NotFoundException
+     * @throws PublicApiClientException
+     */
+    public function testGetConfigs(): void
+    {
+        $response = <<<JSON
+{
+  "status": 200,
+  "data": [
+    {
+      "scope": "role",
+      "name": "purchase something",
+      "state": "enabled"
+    },
+    {
+      "scope": "role",
+      "name": "verify something",
+      "state": "disabled"
+    },
+    {
+      "scope": "authorization",
+      "name": "allow something",
+      "state": "pending"
+    }
+  ]
+}
+JSON;
+
+        $this->httpClient
+            ->expects(self::once())
+            ->method('request')
+            ->with(
+                'get',
+                'https://www.test.com/licenses/XSP1234/configs',
+                [
+                    'headers' => [
+                        'apiKey' => '123456',
+                    ],
+                ]
+            )
+            ->willReturn(new Response(200, [], $response));
+
+        $test = $this->client->getConfigs('XSP1234');
+        $list = iterator_to_array($test);
+        self::assertCount(3, $list);
+
+        /** @var Config $config */
+        $config = array_shift($list);
+        self::assertInstanceOf(Config::class, $config);
+        self::assertSame('role', $config->getScope());
+        self::assertSame('purchase something', $config->getName());
+        self::assertSame('enabled', $config->getState());
+
+        /** @var Config $config */
+        $config = array_shift($list);
+        self::assertInstanceOf(Config::class, $config);
+        self::assertSame('role', $config->getScope());
+        self::assertSame('verify something', $config->getName());
+        self::assertSame('disabled', $config->getState());
+
+        /** @var Config $config */
+        $config = array_shift($list);
+        self::assertInstanceOf(Config::class, $config);
+        self::assertSame('authorization', $config->getScope());
+        self::assertSame('allow something', $config->getName());
+        self::assertSame('pending', $config->getState());
+    }
+
+    /**
+     * @throws GuzzleException
+     * @throws NotFoundException
+     * @throws PublicApiClientException
+     */
+    public function testUpdateConfigRaw(): void
+    {
+        $postData = [
+            Config::COLUMN_SCOPE => 'role',
+            Config::COLUMN_NAME => 'purchaseReservations',
+            Config::COLUMN_STATE => 'enabled',
+        ];
+
+        $config = $this->createMock(Config::class);
```

Comment:
I think you didn't have to mock ```Config``` and use a real ```Config``` instance because it's an entity and is really simple (but no big deal)

Reply:
Done here https://github.com/ArrowSphere/public-api-client/commit/f4e1f96cdd4cc2343d2e971def96b7a677419a49